modules/chat/src/routes/InvitationRoutes.ts

```diff
@@ -0,0 +1,243 @@
+import ConduitGrpcSdk, {
+  ConduitNumber,
+  ConduitRouteActions,
+  ConduitRouteReturnDefinition,
+  ConduitString, GrpcError, ParsedRouterRequest,
+  RoutingManager, UnparsedRouterResponse,
+} from '@conduitplatform/grpc-sdk';
+import { ChatRoom, InvitationToken } from '../models';
+import { isNil } from 'lodash';
+import { status } from '@grpc/grpc-js';
+import * as templates from '../templates';
+
+export class InvitationRoutes {
+  constructor(private readonly grpcSdk: ConduitGrpcSdk, private readonly routingManager: RoutingManager) {
+  }
+
+  async registerTemplates() {
+    this.grpcSdk.config
+      .get('email')
+      .then((emailConfig: any) => {
+        const promises = Object.values(templates).map((template) => {
+          return this.grpcSdk.emailProvider!.registerTemplate(template);
+        });
+        return Promise.all(promises);
+      })
+      .then(() => {
+        console.log('Email templates registered');
+      })
+      .catch((e: Error) => {
+        console.error('Internal error while registering email templates');
+      });
+  }
+
+  declareRoutes() {
+    this.routingManager.route(
+      {
+        path: '/hook/:answer/:invitationToken',
```

Comment:
Make this `/hook/invitations/:answer/:invitationToken` (and update uri in mail templates etc).